webapp/sources/rudder/rudder-web/src/main/javascript/rudder/rudder-datatable.js

```diff
@@ -1402,20 +1401,44 @@ function createNodeTable(gridId, refresh) {
     top.css('bottom', Math.max(bottom.height(), 120));
   });
 
+  await (new Promise(function (resolve, reject) {
+    $.ajax({
+      type: 'GET',
+      url: "/rudder/secure/api/scores/list",
+      success: function (data) {
+        scoreList = data.data
+        resolve(scoreList)
+      },
+      error: function (err) {
+        reject(err)
+      }
+    });
+  }));
+
   var cacheId = gridId + "_columns"
   var cacheColumns = localStorage.getItem(cacheId)
   if (cacheColumns !== null) {
 
     // Filter columns that are null, and columns that have a title that is  not a key in of AllColumns, or if data does not start by software or property
 
     var cache = JSON.parse(cacheColumns).filter(function(c) {
-      return c !== null && (allColumnsKeys.includes(c.title) || (c.data !== undefined && c.title.startsWith("Software")) || c.title.startsWith("Property") )
+      return c !== null 
+        && (
+          allColumnsKeys.includes(c.title) 
+          || (c.data !== undefined && c.title.startsWith("Software")) 
+          || c.title.startsWith("Property")
+          // it is possible that the column data is no longer found in scoresList (e.g. CVE plugin is uninstalled but CVE Score column is saved in cache)
+          || (c.value !== undefined && !!scoreList.find(({ id }) => c.title.endsWith("Score") && c.value === id))
+        )
     })
+    
     columns = cache.map(function(c) {
       if (c.title.startsWith("Property")) {
         return allColumns.Property(c.value,c.inherited);
-      } else { if (c.data.startsWith("software")) {
+      } else { if (c.data?.startsWith("software")) {
         return allColumns.Software(c.value);
+      } else if (/^.+\sScore$/.test(c.title)) {
+        return allColumns["Score details"](c.value);
```

Comment:
Specifically parse the score details columns from the localstorage, it was just forgotten in previous PRs for score details